acceptance: do not print uninteresting error

tbg opened the pull request:
<!-- Reviewable:start -->

This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/7909)

<!-- Reviewable:end -->

tbg committed:
acceptance: do not print uninteresting error

petermattis commented on acceptance/allocator_terraform/main.tf:
```diff
@@ -13,6 +13,7 @@ resource "google_compute_instance" "cockroach" {
   disk {
     image = "${var.gce_image}"
     size = "${var.cockroach_disk_size}" # GB
+    # type    = "pd-ssd" # uncomment for persistent SSD
```
Can this be a flag passed to the test? Seems like the `cockroach_binary` terraform var can be set via the `--at.cockroach-binary` flag.

tschottdorf replied:
It can be plumbed, but I don't see the necessity at this point. Do you think it's worth it?

petermattis replied:
No strong reason. I'm just imagining others will want to run the test with `pd-ssd` disks.

tschottdorf replied:
Ok, done.

On Tue, Jul 19, 2016 at 9:23 AM Peter Mattis <notifications@github.com>
wrote:

> In acceptance/allocator_terraform/main.tf
> <https://github.com/cockroachdb/cockroach/pull/7909#discussion_r71336286>:
>
> > @@ -13,6 +13,7 @@ resource "google_compute_instance" "cockroach" {
> >    disk {
> >      image = "${var.gce_image}"
> >      size = "${var.cockroach_disk_size}" # GB
> > +    # type    = "pd-ssd" # uncomment for persistent SSD
>
> No strong reason. I'm just imagining others will want to run the test with
> pd-ssd disks.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/7909/files/2f1fa4ba848d593fb1998bd7f98e0cf34390fd05#r71336286>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AE135ExWwH-pc9ZwrE1oqbbLvffc3Aq4ks5qXM-9gaJpZM4JPnbk>
> .
>
-- 

-- Tobias

petermattis commented:
LGTM

tbg committed:
acceptance: make GCE disk type configurable

Saves the next person the lookup when they want to run a cluster
with SSD instead. We should be using spinny disk for the normal
case as that makes expensive compactions much more painful, and
we do want to know when that happens.

petermattis commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Review status: 0 of 5 files reviewed at latest revision, all discussions resolved, some commit checks pending.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/cockroach/7909#-:-KN3U1MY6LmVQmWoTDHs:1417088721)*
<!-- Sent from Reviewable.io -->
